.gitignore

```diff
@@ -7,3 +7,4 @@
 /id_rsa
 /rp
 /secrets
+/pyenv
```

Comment:
Please add *.pyc and ensure that you remove CPython compilation artifacts from source control.